src/main/java/echoquery/intents/CountHandler.java

```diff
@@ -10,82 +8,57 @@
-
-      // TODO(vqtran): Pull out this parsing into a reusable Parser class that
-      // parses it into some sort of abstract syntax / handles error checking.
-      String query = "SELECT COUNT(*) FROM " + table;
-      String match = null;
-      if (column != null) {
-        match = (colVal == null) ? colNum : colVal;
-        String comparison = "";
-        if (equals != null) {
-          comparison = "=";
-        }
-        if (greater != null) {
-          comparison = ">";
-        }
-        if (less != null) {
-          comparison = "<";
-        }
-        query += " WHERE " + column + " " + comparison + " '" + match + "'";
-      }
-
-      Statement statement = SingletonConnection.getInstance().createStatement();
-      ResultSet result = statement.executeQuery(query);
-      result.first();
+      Query query = QueryBuilder.from(intent).build();
+      int result = querier.execute(query);
 
       // TODO(vqtran): Pull this out into a response generator class that takes
       // the result and the Parser's abstract syntax object.
-      String response = "There ";
-      if (result.getInt(1) == 1) {
-        response +=
-            "is " + TranslationUtils.convert(result.getInt(1)) + " row";
-      } else {
-        response +=
-            "are " + TranslationUtils.convert(result.getInt(1)) + " rows";
-      }
-      response += " in the " + table + " table";
-      if (column != null) {
-        response += " where the value in the " + column + " column is ";
-        if (equals != null) {
-          response += "equal to ";
-        }
-        if (greater != null) {
-          response += "greater than ";
-        }
-        if (less != null) {
-          response += "less than ";
-        }
-        if (colVal == null) {
-          response += TranslationUtils.convert(Integer.parseInt(match));
-        } else {
-          response += colVal;
-        }
-      }
-      response += ".";
+//      String response = "There ";
+//      if (result.getInt(1) == 1) {
+//        response +=
+//            "is " + TranslationUtils.convert(result.getInt(1)) + " row";
+//      } else {
+//        response +=
+//            "are " + TranslationUtils.convert(result.getInt(1)) + " rows";
+//      }
+//      response += " in the " + table + " table";
+//      if (column != null) {
+//        response += " where the value in the " + column + " column is ";
+//        if (equals != null) {
+//          response += "equal to ";
+//        }
+//        if (greater != null) {
+//          response += "greater than ";
+//        }
+//        if (less != null) {
+//          response += "less than ";
+//        }
+//        if (colVal == null) {
+//          response += TranslationUtils.convert(Integer.parseInt(match));
+//        } else {
+//          response += colVal;
+//        }
+//      }
+//      response += ".";
 
-      return Response.say(response);
+      return Response.say(result + "");
```

Comment:
why not just result.toString()? that seems clearer this looks like javascript style